### ISSUE TITLE
a11y issues with 3 dot share on Vega maps

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1043]
Explore relationships between x and y in z	Orientation	[A11Y - Health Equity Tracker]: Options button not rendering properly when user changes page orientation to portrait mode_Web_1.18(a)	"ENVIRONMENT:
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Test on a chromebook with a rotating hinge

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Click on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
3). Observe the page in landscape mode
4). Change the orientation to portrait.
5). Observe the page in portrait mode

EXPECTED RESULT:
Page content is displayed properly and alike in both orientation.
 
OBSERVED RESULT:
Options button besides some choropleth map is not rendering properly in portrait mode. A little portion in getting truncated in three instances.

Replication video attached"	https://drive.google.com/file/d/1BMXbPqsAlukwzC7vYigFAtEVURUDCXsp/view?usp=sharing	Same issue is observed on Compare rates page.
Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: Options button besides choropleth map in Explore relationships page doesn't have a label_Web_1.1	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Turn on the respective screen reader

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Navigate to 'Diabetes cases per 100K' & 'Total Covid-19 cases per 100K' section on the page
5). Navigate to the options button besides choropleth map
6). Observe the announcement.

EXPECTED RESULT:
Button should a label and it should be properly announced by screen reader.
 
OBSERVED RESULT:
Voiceover announces the option button as collapsed, disclosure triangle. Chromevox announces the option button only as collapsed.

Replication video attached"	https://drive.google.com/file/d/1DrBa-abevLUYTQ7gSCDrhWO9rcTnxdoj/view?usp=sharing	"Same issue is observed on Compare rates page.

Announced as collapsed, disclosure triangle graphic in Android

Device: Pixel 4
Android Version: 11
Chrome Browser Version: 91.0.4472.101"
Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: User unable to access options button besides choropleth map by linear navigation_Web_1.3	"ENVIRONMENT:
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Turn on the respective screen reader

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Navigate to 'Diabetes cases per 100K' & 'Total Covid-19 cases per 100K' section on the page
5). Try linear navigating to the options button besides choropleth map
6). Observe the behavior.

EXPECTED RESULT:
User should be able to reach and perform action on any element on the page using linear navigation.
 
OBSERVED RESULT:
User is unable to reach the options button besides choropleth map by linear navigation using chromevox. User is able to access the button using tab navigation.

Replication video attached"	https://drive.google.com/file/d/1e5oh0_yC37_MI_Nup4QGiNneqjjJuCse/view?usp=sharing	Same issue is observed on Compare rates page.
Explore relationships between x and y in z	Large Text	[A11Y - Health Equity Tracker]: Options button not rendering properly when large text has been enabled_Web_1.16(b).	"ENVIRONMENT:
Platform: macOS 11.3
Browser: Chrome Version 90.0.4430.212 (Official Build) (x86_64)

Pre Req:
Enable Large Text (Browser zoomed to 200%)

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Click on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Observe the items on the page when large text has been enabled.

EXPECTED RESULT:
Page content should render properly when large text has been enabled.

OBSERVED RESULT:
Options button besides choropleth map is not rendering properly when large text has been enabled.

Screenshot attached"	"https://drive.google.com/file/d/1Gg2EzJz9wSCh6cOneXuVFuVzeHN9LuWe/view?usp=sharing

https://drive.google.com/file/d/15Iic-3vRYDRAjvfbNETZtxof_bCYkMNn/view?usp=sharing"	Same issue is observed on Compare rates page.

Environment: Web: Chromebook (Chrome OS version: 91.0.4456.0 (official build) dev 64 bit ), Chromevox Macbook (macOS Big sur: 11.3, Chrome browser: 89.0.4430.212 (official build) x86_64), Voiceover Windows Windows 10 Enterprise, Chrome browser:Version 90.0.4430.212 (Official Build) (64-bit), JAWS (version:2019.1912.1 ILM) / NVDA Prerequisite: User is a keyboard user. Priority / Severity : P2 / S2 Steps: 1. Navigate to URL: https://dev.healthequitytracker.org/exploredata?mls=1.covid..3.00&mlp=disparity 2. TAB navigate within the page onto the Explore the data --> Investigate rates of Covid-19 in United States. 3. TAB navigate within the page to reach the more options (3 vertical dots) available under any of the following section header 'COVID-19 cases per 100k people' OR 'Unknown Race and Ethnicity for Share of total COVID-19 cases' OR 'Share of total COVID-19 cases vs Share of Population by Race and Ethnicity in United States' OR 'COVID-19 cases per 100k people by Race and Ethnicity in United States' 4. Observe upon keyboard TAB focus is set on the more options (3 vertical dots) button, there is no visual indicator like a border or highlight. Actual result: 1. Observe upon TAB navigate to the control 'more options' , there is no visual indicator like border or highlight being displayed. This will confuse the user as user will not know/understand where the current focus is set. upon click enter , the dropdown list is displayed indicating the focus was set on the more options button. Note: no actionable control available on the toast. Expected result: Confirm that a visual indicator, like a border or highlight, is present for each element that currently has the keyboard focus. Screenshot / video attachment: https://screencast.googleplex.com/embed/NDg0NzgyNTA5NzU4ODczNnw1NTRiZDhkYy0yYg"
